### PR TITLE
helidon formula

### PR DESCRIPTION
**Pull request by tvallin**
Note to myself:
- Change the url to the helidon one when published.

### CODE REVIEW

File: Formula/helidon.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Helidon < Formula
+  desc "Set of Java libraries for writing microservices that run on Netty"
```
Comment (romain-grecourt) [inline Formula/helidon.rb]: The Helidon CLI lets you easily create a Helidon project by picking from a set of archetypes. It also supports a developer loop that performs continuous compilation and application restart, so you can easily iterate over source code changes.

File: Formula/helidon.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "https://helidon.io/"
+  url "file:///Users/tvallin/workspace/helidon-build-tools/cli/impl/target/distribution/helidon.zip"
```
Comment (romain-grecourt) [inline Formula/helidon.rb]: We need to stage the distribution in helidon.io ; that means a change in `helidon-site-files`.
We can reference the Maven Central URL now: https://repo1.maven.org/maven2/io/helidon/build-tools/cli/helidon-cli-impl/3.0.4/helidon-cli-impl-3.0.4.zip

**Review by romain-grecourt — COMMENTED** [review]
Initial review

File: Formula/helidon.rb (hunk 1 above)
Comment (tvallin) [reply Formula/helidon.rb]: HomeBrew policy: Description is too long, it should be less than 80 character and it cannot start with an article nor formula name.

File: Formula/helidon.rb (hunk 2 above)
Comment (tvallin) [reply Formula/helidon.rb]: Had to change the url because of homebrew complaining, but it works fine.

File: Formula/helidon.rb (hunk 1 above)
Comment (romain-grecourt) [reply Formula/helidon.rb]: "Command line tool for Helidon application development".

File: Formula/helidon.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class Helidon < Formula
+  desc "Command line tool for Helidon application development"
+  homepage "https://helidon.io/"
+  url "https://search.maven.org/remotecontent?filepath=io/helidon/build-tools/cli/helidon-cli-impl/3.0.4/helidon-cli-impl-3.0.4.zip"
```
Comment (barchetta) [inline Formula/helidon.rb:4]: When I try this URL behind the Oracle proxy (using browser or curl) I get a 403.

 I also found this: https://central.sonatype.org/faq/what-happened-to-search-maven-org/ 

Have you checked with homebrew why they don't like https://repo1.maven.org/maven2/io/helidon/build-tools/cli/helidon-cli-impl/3.0.4/helidon-cli-impl-3.0.4.jar ?

File: Formula/helidon.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  version "3.0.0"
```
Comment (barchetta) [inline Formula/helidon.rb]: Why 3.0.0 and not 3.0.4 ?

File: Formula/helidon.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  # change to final sha256
+  sha256 "4eadf18d977c18858043700ea4745628c9620a73d96d47a6a4ba0a9fa407cdec"
+  license "Apache-2.0"
+
+  def install
+    mkdir_p libexec/"bin"
+    mv "bin/helidon", libexec/"bin/helidon"
+    mv "lib", libexec/"lib"
+    (bin/"helidon").write_env_script libexec/"bin/helidon", Language::Java.overridable_java_home_env("17")
```
Comment (barchetta) [inline Formula/helidon.rb]: Not sure of the consequences of "17" here. From what I can tell the CLI will run with 11. Assuming that is correct then we do not want to require 17.

File: Formula/helidon.rb (hunk 3 above)
Comment (tvallin) [reply Formula/helidon.rb:4]: See the following: https://github.com/orgs/Homebrew/discussions/4276.

**Pull request closed without merging**